main/res/values/strings.xml

```diff
@@ -1590,5 +1596,8 @@
     <string name="init_low_disk_space_message">The geocache data directory is running low on storage space. This can cause malfunctions of c:geo. Please free up some space.</string>
     <string name="init_confirm_debug">c:geo is running in debug mode!</string>
     <string name="list_confirm_debug_message">The debug mode causes instabilities! Do you want to deactivate the debug mode?</string>
+    <string name="resault">Resault:</string>
+    <string name="equation_hint">a^2 + b^2</string>
```

Comment:
That's not a bad idea and I did think about it.

In the end I decided to go for the 'Pythagoras-esk' formula as it looks neater and if the user understands the '^' operator is available (along with the '+') they will probably naturally assume the '-', '*' and '/' are too.

The operator that could easily be missed is the '%' operator but for the sake of neatness I decided to leave it out.  The important thing to communicate is the fact that expressions can be entered and to hint at the (lower-case) 'free variable' feature.  The 'power' operator was just thrown in for good measure because it was there, and as for the '%' operate... I only added that so I could make the "base 13" joke (which I expect very few people will get, but those that do will hopefully appreciate it ;))